src/components/DpSearchField/DpSearchField.vue

```diff
@@ -38,6 +38,7 @@ export default {
 
     /**
      * Value has to be a css class like 'u-1-of-2'
+     * @deprecated Apply inputWidth to the parent element of DpSearchField.
```

Comment:
Applying inputWidth to the parent element will not have the same effect, however it will be removed. There is no guidance here, except "do not use it anymore".

```suggestion
     * @deprecated Will be removed when the component layout is refactored.
```